tests/cephadm/test_legacy_services.py

```diff
@@ -0,0 +1,51 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify if the cephadm shell tries to infer config from
+    a dummy mon created.
+    Verifies bz:2080242
+
+    Args:
+        ceph_cluster: Ceph cluster object
+        **kw :     Key/value pairs of configuration information to be used in the test.
+
+    Returns:
+        int - 0 when the execution is successful else 1 (for failure).
+    """
+
+    config = kw.get("config")
+
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+
+    # Create dummy mon on all non-mon nodes
```

Comment:
you added `break` now, so it won't execute on all non-mon nodes,
can you correct this comment and attach new test run logs?

Reply:
@sunilangadi2 Noted, will fix the comment.
New logs are attached with the changes suggested earlier are attached above.
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-NJRI44/

Reply:
In the logs i can see that it's testing on more than one non monitor node, are you sure these are logs after adding break?

Reply:
@sunilangadi2 yes I could see multiple nodes been tested. I'll re-run and upload the results here

Reply:
@sunilangadi2 attaching the full log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-YZANU0/
Now the check breaks after checking on a non-mon node